docs/replatforming.md

```diff
@@ -84,6 +84,8 @@ We want to avoid commiting the `assetPaths.json` file, but need to keep it out o
 1. Open up `.git/info/exclude`
 2. Add `assetPaths.json` to that file
 
+If that doesn't work, type in `git update-index --assume-unchanged _data/assetPaths.json` into the terminal.
```

Comment:
hm... let's keep any eye on this one. If it didn't work my hunch is that it got committed at some point and need to be removed with `git rm` but TBD!

Reply:
Not sure how it happened either, I was staring at the `assetPaths.json` in `/.git/info/exclude` and even added `_data/assetPaths.json` but GitHub Desktop was still showing it! So I Googled a solution. 😅

Reply:
To be explicit, I don't think you did anything incorrectly, I am more wondering if we let it slip in somewhere else I saw it show up recently too. Since it's keeps getting regenerated on every build, I think we can safely use `git rm` on it